<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YksityistilienPaattaja</class>
 <widget class="QDialog" name="YksityistilienPaattaja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yksityistilien päättäminen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="browser"/>
   </item>
   <item>
    <widget class="QCheckBox" name="paataTilit">
     <property name="text">
      <string>Päätä yksityistilit peruspääomaan</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="paataTulos">
     <property name="text">
      <string>Päätä tilikauden tulos peruspääomaan</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Peruspääoman tili</string>
       </property>
       <property name="buddy">
        <cstring>tiliCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TiliCombo" name="tiliCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TiliCombo</class>
   <extends>QComboBox</extends>
   <header location="global">tools/tilicombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>YksityistilienPaattaja</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>YksityistilienPaattaja</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
